qiskit/transpiler/passes/routing/sabre_swap.py

```diff
@@ -319,159 +313,114 @@ def process_dag(block_dag, wire_map):
 
 
 def _apply_sabre_result(
-    mapped_dag,
-    root_dag,
-    qubit_indices,
-    canonical_register,
-    initial_layout,
+    out_dag,
+    in_dag,
     sabre_result,
+    initial_layout,
+    physical_qubits,
     circuit_to_dag_dict,
-    component_map=None,
 ):
-    bit_to_qreg_idx = {bit: idx for idx, bit in enumerate(canonical_register)}
+    """Apply the ``SabreResult`` to ``out_dag``, mutating it in place.  This function in effect
+    performs the :class:`.ApplyLayout` transpiler pass with ``initial_layout`` and the Sabre routing
+    simultaneously, though it assumes that ``out_dag`` has already been prepared as containing the
+    right physical qubits.
+
+    Mutates ``out_dag`` in place and returns it.  Mutates ``initial_layout`` in place as scratch
+    space.
+
+    Args:
+        out_dag (DAGCircuit): the physical DAG that the output should be written to.
+        in_dag (DAGCircuit): the source of the nodes that are being routed.
+        sabre_result (SabreResult): the result object from the Rust run of the Sabre routing
```

Comment:
This isn't actually a sabre result object is it? Isn't the tuple of: `(SwapMap, gate_order, NodeBlockResults)`. It was a sabre result before #10652